build.assets/Makefile

```diff
@@ -156,9 +157,10 @@ buildbox-fips: buildbox-centos7-fips
 buildbox-centos7:
 	$(REQUIRE_HOST_ARCH)
 	$(call PULL_ON_CI,$(BUILDBOX_CENTOS7))
-	DOCKER_BUILDKIT=1 docker build \
```

Comment:
Should we just get rid `DOCKER_BUILDKIT=1` throughout the repo? It looks like it's not necessary for `docker buildx`, and that `docker buildx` is the new standard way to run docker builds:

https://github.com/moby/buildkit/issues/1214#issuecomment-1763493367

Reply:
This PR does exactly that. The only `DOCKER_BUILDKIT` remaining after this is in Drone (`.drone.yml` and `dronegen/container_image_products.go`), and I'm not touching Drone stuff, as it's going away anyway. :-)